metricbeat/module/system/process/helper.go

```diff
@@ -336,7 +339,7 @@ func (procStats *ProcStats) GetProcStats() ([]common.MapStr, error) {
 		return nil, err
 	}
 
-	processes := []common.MapStr{}
+	processes := []Process{}
```

Comment:
It's best to declare the empty slice as `var processes []Process` because it avoids the allocation if the slice goes unused. https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices